src/app/csv_parser.py

```diff
@@ -0,0 +1,24 @@
+import sqlite3
```

Comment:
For the future - Worth to move DB configuration to 'set_databese.py`, and all db operations to some db repository or just `db_connection.py`  module. Then just call methods like:
```python 
from db_connection import get_session
db_session = get_session()
```